src/sonic-config-engine/minigraph.py

```diff
@@ -66,8 +66,10 @@ def get_peer_switch_info(link_metadata, devices):
         if "PeerSwitch" in data:
             peer_hostname = data["PeerSwitch"]
             peer_lo_addr = devices[peer_hostname]["lo_addr"] 
+            peer_lo_addr = ipaddress.ip_network(UNICODE_TYPE(peer_lo_addr))
```

Comment:
if the prefix change for any reason in the minigraph, this will zero out the host section of the IP address. I think it is better to check for if the prefix length is 32. if not, you strip `/<prefix len>` section. Or err out if it is not `/32`

Reply:
I thought that we needed to remove the prefix length completely because our config DB schema is expecting a plain address without the prefix length. @prsunny can you confirm?

Reply:
@tahmed-dev , this is ip address field, so we can safely ignore the subnet part. Ideally we expect a `/32` here, however it is still ok to extract the address part even if not `/32`